
autoexplorer:
    debug_data_save_path:   '/home/hankm/results/autoexploration'
    costmap_conf_thr:       0.4 #0.5 (corridor) # 0.4 (fetch)   # the higher the more strict  
    gridmap_conf_thr:       0.0 # 0.6 (fetch)   # 0.8 for (gmapping)            # the higher the more strict
    occupancy_thr:          40      # 0 ~ 100
    lethal_cost_thr:        1   # 0 ~ 100  uk= -1      # the lower the more strict 
    global_width:           4000 #8000
    global_height:          4000 #8000
    weak_comp_thr:          5
    num_downsamples:        0
    unreachable_decision_bound:   0.2 # < 4 * 0.05 res >
